components/atoms/PolygonIcon.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+
```

Comment:
please make a PolygonIcon (and other SVGs) a plain constant to make it clearer, e.g.
```typescript
const PolygonIcon = (
   <svg fill="white" ...
   ...
   </svg>
)
```